odl/solvers/functional/functional.py

```diff
@@ -1268,6 +1333,186 @@ def __str__(self):
         return '{}.convex_conj'.format(self.convex_conj)
 
 
+class BregmanDistance(Functional):
+    """The Bregman distance functional.
```

Comment:
Perhaps a few non-technical words on what this does below this would help, e.g. (copy-paste from wikipedia) "The Bregman divergence or Bregman distance is similar to a metric, but satisfies neither the triangle inequality nor symmetry."